### ISSUE TITLE
Eskhandar not being 3% chance

### ISSUE THREAD

**Issue by lyssebabz** [slowtorta/turtlewow-bug-tracker#5719]
![Screenshot from 2023-12-22 21-41-39](https://github.com/slowtorta/turtlewow-bug-tracker/assets/20995435/9d0d922b-4a15-4465-993c-32fb6cdd1447)
![Screenshot from 2023-12-22 21-41-20](https://github.com/slowtorta/turtlewow-bug-tracker/assets/20995435/89599f24-873e-4244-9c39-1282fa6d7940)
https://forum.turtle-wow.org/viewtopic.php?p=52795&hilit=eskhandar#p52795

**Comment by twowTurtReynolds** [comment]
I set my haste to rediculous speeds and crit to 100%, ran procwatch, and it looks like we're getting a proc every 35 hits, so 3%. Rest assured the item works as intended.

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/151180101/ff80bce9-ef17-4928-8da4-ba093531d741)

We just need to update the text to be correct.

**Comment by lyssebabz** [comment]
> I set my haste to rediculous speeds and crit to 100%, ran procwatch, and it looks like we're getting a proc every 35 hits, so 3%. Rest assured the item works as intended.
> 
> ![image](https://private-user-images.githubusercontent.com/151180101/292697687-ff80bce9-ef17-4928-8da4-ba093531d741.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM1MDM0NTksIm5iZiI6MTcwMzUwMzE1OSwicGF0aCI6Ii8xNTExODAxMDEvMjkyNjk3Njg3LWZmODBiY2U5LWVmMTctNDkyOC04ZGE0LWJhMDkzNTMxZDc0MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNVQxMTE5MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZWEwYWE3ZDA3YTVlYTg0MmJkYjRkMDVhYWMzYzJjZGI5MTY4YWZlNzBlZmI0MzI3MzBjZDBkNzk3OTg0M2EyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ZWr3s1o8ZS1y6OuUzn291QVBXNWFhibSym3mLACn9IQ)
> 
> We just need to update the text to be correct.

Thanks for checking it out and that looks so fun!

**Comment by twowTurtReynolds** [comment]
No, please keep it open. This report is now for the developers to update the text.

**Comment by lyssebabz** [comment]
I'm sorry!
